### PR TITLE
TRSv2-398 - Change accessibility statement

### PR DESCRIPTION
**Pull request by nahmed99**

### CODE REVIEW

**Review by yusufsheiqh — COMMENTED** [review]
Unsure if this page is using the GOV.UK Design System. For example, it does not make sense to use `terms-and-conditions-page` for an accessibility statement.

**Comment by nahmed99** [comment]
> Unsure if this page is using the GOV.UK Design System. For example, it does not make sense to use `terms-and-conditions-page` for an accessibility statement.

Not sure either. Yep, I agree it doesn't make sense, but that was in the original code and I couldn't think what to replace it with.

File: trade_remedies_public/templates/registration/accessibility_statement.html [hunk 1]
```diff
@@ -4,72 +4,131 @@
...
-            Accessibility statement for the {% organisation_initialism %} service
-        </h1>
         <div class="column-two-thirds left-column">
-
-            <h2 class="heading-medium">Using this website</h2>
-            <p>This website is run by the {% organisation_name %} ({% organisation_initialism %}), part of Department
-                for International Trade (DIT). We want as many people as possible to use this website, so you should be
-                able to:</p>
+            <h2 class="heading-medium">Accessibility statement</h2>
+            <p>This statement applies to content published on <a href="www.traderemediesservice.gov.uk">www.traderemediesservice.gov.uk</a>.</p>
```
Comment (yusufsheiqh) [inline trade_remedies_public/templates/registration/accessibility_statement.html:9]: This is a dead link.

```suggestion
            <p>This statement applies to content published on <a href="https://www.trade-remedies.service.gov.uk">www.trade-remedies.service.gov.uk</a>.</p>
```

File: trade_remedies_public/templates/registration/accessibility_statement.html [hunk 2]
```diff
@@ -4,72 +4,131 @@
...
+
+            <h2 class="heading-medium">What we’re doing to improve accessibility</h2>
+            <p>We are fixing content which fails to meet the Web Content Accessibility Guidelines version 2.1. We will 
+                update this page when issues are fixed.</p>
+
+
+            <h2 class="heading-medium">How we tested this website</h2>
+            <p>This website was last tested on 16 Apr 2019. The test was carried out by the 
+                <a href="https://digitalaccessibilitycentre.org/">Digital Accessibility Centre</a>. 
+                You can <a href="https://dbis.sharepoint.com/sites/TRSTeam/Shared%20Documents/Forms/AllItems.aspx?csf=1&web=1&e=0pr8JL&cid=bcaf86ea%2D943b%2D4ce2%2D8fb2%2Dfe4a85c67c18&FolderCTID=0x012000FEEC8DEF492D1244A62BBA3A60EE16CF&id=%2Fsites%2FTRSTeam%2FShared%20Documents%2FGeneral%2FAccessibility%2FTRS%20Accessibility%20Audit%202019%2FDepartment%20for%20International%20Trade%2DUK%20Trade%20Remedies%20Authority%5FAccessibility%20Report%5FJan%202019%5FWCAG%202%2E1%5B1%5D%5B33%5D%2Epdf&parent=%2Fsites%2FTRSTeam%2FShared%20Documents%2FGeneral%2FAccessibility%2FTRS%20Accessibility%20Audit%202019">
```
Comment (yusufsheiqh) [inline trade_remedies_public/templates/registration/accessibility_statement.html:119]: I think this should be a static file if it is meant to be available to the general public.

File: trade_remedies_public/templates/registration/accessibility_statement.html (hunk 1 above)
Comment (nahmed99) [reply trade_remedies_public/templates/registration/accessibility_statement.html]: Done.

File: trade_remedies_public/templates/registration/accessibility_statement.html (hunk 2 above)
Comment (nahmed99) [reply trade_remedies_public/templates/registration/accessibility_statement.html]: Link removed.

**Review by yusufsheiqh — COMMENTED** [review]
One minor comment about the date, otherwise looks good to merge.

File: trade_remedies_public/templates/registration/accessibility_statement.html [hunk 3]
```diff
@@ -4,72 +4,129 @@
...
-                </div>
-            </div>
+            <p>We’ll consider your request and get back to you in 5 days.</p>
+
+            <h2 class="heading-medium">Reporting accessibility problems with this service</h2>
+            <p>We’re always looking to improve the accessibility of this service. If you find any problems 
+                that are not listed on this page or think we're not meeting accessibility requirements:</p>
+            <ul class="list list-bullet">
+                <li>email <a href="mailto: contact@traderemedies.gov.uk">contact@traderemedies.gov.uk</a></li> {# /PS-IGNORE #}
+                <li>call +44 (0) 20 7215 5000</li>
```
Comment (chris-pettinga) [inline trade_remedies_public/templates/registration/accessibility_statement.html:49]: Shall we make this a clickable link?

```suggestion
               <li><a href="tel:+442072155000">call +44 (0) 20 7215 5000</a></li>
```

File: trade_remedies_public/templates/registration/accessibility_statement.html [hunk 4]
```diff
@@ -4,72 +4,129 @@
...
+            <p>We are fixing content which fails to meet the Web Content Accessibility Guidelines version 2.1. We will 
+                update this page when issues are fixed.</p>
+
+
+            <h2 class="heading-medium">How we tested this website</h2>
+            <p>This website was last tested on 16 Apr 2019. The test was carried out by the 
+                <a href="https://digitalaccessibilitycentre.org/">Digital Accessibility Centre</a>. 
+
+            <h2 class="heading-medium">Preparation of this accessibility statement</h2>
+            <p>This statement was prepared on 3 Jul 2019 and updated on 15 Dec 2022.</p>
```
[the remark this answers is not in the document]
Comment (nahmed99) [reply trade_remedies_public/templates/registration/accessibility_statement.html:121]: Leave it - as that is when the statement was probably updated. Won't get published till later, I expect.

**Review by chris-pettinga — APPROVED** [review]

**Review by yusufsheiqh — APPROVED** [review]

File: trade_remedies_public/templates/registration/accessibility_statement.html (hunk 3 above)
Comment (yusufsheiqh) [reply trade_remedies_public/templates/registration/accessibility_statement.html]: Resolved in https://github.com/uktrade/trade-remedies-public/pull/244/commits/b9acf08f1b828334803a23ceb7d8ce1a4dd8380a.

**Pull request merged**